Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/Tools/MigrationWindow.cs

```diff
@@ -0,0 +1,417 @@
+
+                var selectionType = selectedToolbar == ToolbarOption.Objects ? typeof(GameObject) : typeof(Object);
+                var allowSceneObjects = selectedToolbar == ToolbarOption.Objects;
+
+                selection = EditorGUILayout.ObjectField(null, selectionType, allowSceneObjects);
+                if (selection)
+                {
+                    AddSelection();
+                }
+            }
+            EditorGUILayout.Space();
+
+            using (new EditorGUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                DrawMigrationObjectsList();
+            }
+        }
+
+        private void DrawProjectToolbar()
+        {
+            EditorGUILayout.Space();
+            using (new EditorGUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+                {
+                    if (GUILayout.Button("Migrate"))
+                    {                        
+                        MigrateAllPrefabs();
+                        MigrateAllScenes();
+                    }
+
+                    using (new EditorGUILayout.HorizontalScope())
+                    {
+                        GUILayout.FlexibleSpace();
+
+                        string tooltip = "All Scenes and Prefabs Selected";
+                        EditorGUILayout.LabelField(new GUIContent(tooltip, InspectorUIUtility.WarningIcon));
+                    }
+                }
+            }
+        }
+
+        private void DrawMigrationObjectsList()
+        {
+            scrollPosition = EditorGUILayout.BeginScrollView(scrollPosition);
+
+            foreach (var migrationObject in migrationObjects)
+            {
+                using (var horizontal = new GUILayout.HorizontalScope())
+                {
+                    using (new EditorGUI.DisabledGroupScope(true))
+                    {
+                        EditorGUILayout.ObjectField(migrationObject, typeof(Object), false);
+                    }
+                    var icon = EditorGUIUtility.IconContent("winbtn_win_min_h");
+
+                    if (GUILayout.Button(icon, GUILayout.Width(30)))
+                    {
+                        migrationObjects.Remove(migrationObject);
+                        break;
+                    }
+                }
+            }
+            EditorGUILayout.Space();
+
+            using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                if (migrationObjects.Count == 0)
+                {
+                    GUI.enabled = false;
+                }
+
+                if (GUILayout.Button("Migrate"))
+                {
+                    MigrateSelection();
+                }
+
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    if (migrationObjects.Count > 0)
```

Comment:
Super nit: Thoughts on having the HorizontalScope enclosed by the if statement here, rather than the other way around (i.e. how it is now)?

Or do we need the HorizontalScope here even if it's empty?

Reply:
Changing this.